sdk/src/calibration_96tof1.cpp

```diff
@@ -97,8 +108,129 @@ aditof::Status Calibration::displayCalMap() const {
     return Status::OK;
 }
 
-aditof::Status Calibration::getAfeFirmware(const std::string &mode,
-                                           std::vector<uint16_t> &data) const {
+//! SaveCalMap - Save the entire calibration map
+/*!
+SaveCalMap - Saves the entire calibration map as binary to a file.
+\param calibration_map - Calibarion map to be displayed
+\param file_name - File where the calibration map needs to be stored
+*/
+aditof::Status
+Calibration96Tof1::saveCalMap(std::shared_ptr<aditof::DeviceInterface> device) {
+    using namespace aditof;
+
+    std::vector<float> data;
+    for (const auto &mapElement : m_calibration_map) {
+        data.push_back(mapElement.first);
+        const packet_struct &sub_packet_map = mapElement.second;
+        data.push_back((float)sub_packet_map.size);
+
+        for (const auto &packet : sub_packet_map.packet) {
+            data.push_back(packet.first); // write parameter key
+            data.push_back(
+                (float)packet.second.size); // write size of parameter
+
+            for (const auto &value : packet.second.value) {
+                data.push_back(
+                    static_cast<float>(value)); // write parameter values
+            }
+        }
+    }
+
+    float size = static_cast<float>(data.size() * sizeof(uint32_t));
+    device->writeEeprom((uint32_t)0, (uint8_t *)&size, (size_t)4);
+    device->writeEeprom((uint32_t)4, (uint8_t *)data.data(), (size_t)size);
+
+    return Status::OK;
+}
+
+//! ReadCalMap - Read the entire calibration map
+/*!
+ReadCalMap - Read the entire calibration map from a binary file
+\param calibration_map - Calibarion map to be read
```

Comment:
doc for parameters does not match the actual parameters

Reply:
Done